packages/server/tests/localAcceptance.env

```diff
@@ -14,3 +14,4 @@ HAPI_CLIENT_TRANSACTION_RESET=50
 HAPI_CLIENT_DURATION_RESET=3600000
 HAPI_CLIENT_ERROR_RESET=[50]
 FILTER_API_ENABLED=true
+ETH_POPULATE_SYNTHETIC_CONTRACT_RESULTS=true
```

Comment:
nit: is this not the default? If so why is this needed?

Reply:
I think the default is false. At least that is what is in `~/hedera-json-rpc-relay/docs/configuration.md` And since you asked. I saw in `eth.ts` something strange.

>   private readonly shouldPopulateSyntheticContractResults = process.env.ETH_POPULATE_SYNTHETIC_CONTRACT_RESULTS
    ? process.env.ETH_POPULATE_SYNTHETIC_CONTRACT_RESULTS === 'true'
    : false;
    
Do you think it's a mistake ?

Reply:
It's not a mistake, we've decided to keep it false by default and configure it depending on needs, until it's clear that it won't cause any troubles for normal ethereum receipts.